### ISSUE TITLE
✨ Feature: Add status bar or other indicator when sending lots of assets to show the progress

### ISSUE THREAD

**Issue by atrefonas** [algodex/algodex-mailbox#138]
# ℹ Overview

When you send to lots of addresses it takes a while. There should be some sort of status indicator.

The status should be broken up into 4 stages: 
1) Looking up wallet balances & signing logical signatures
2) Sending transactions to the MyAlgo Connect bridge for signing
3) waiting for user signature via the MyAlgo GUI
4) sending the transactions to the recipients on the blockchain

**Comment by atrefonas** [comment]
@stephclarkga will work on a design

**Comment by stephclarkga** [comment]
Here's the Figma file with the Progress Bar broken into the 4 different steps: https://www.figma.com/file/re7P6cPZrZHWBFN4SxozOy/Algodex-Mailbox-Interface?node-id=2%3A3156

All steps use a determinate linear progress bar, except the 3rd step when waiting for the MyAlgo signature. That one is indeterminate and keeps moving until the system verifies the transactions have been signed by MyAlgo and moves onto the next step and sends the transactions.

I added a sentence on the last page of the flow below the link user's can share to redeem assets to explain that receivers already opted in don't need to use this link but I think I saw a GitHub issue in progress where a checkbox is being added so that could change how we present that information and we can discuss that one further. I think it's important to highlight that fact though.
